sql-statements/sql-statement-drop-stats.md

```diff
@@ -12,10 +12,10 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-drop-stats/']
 
 ```ebnf+diagram
 DropStatsStmt ::=
-    'DROP' 'STATS' TableName
+    "DROP" "STATS" TableNameList 
```

Comment:
It looks like single quote `'` should be used instead of double quote `"`.

Reply:
Yes! Fixed.